pytest/tests/mocknet/release_forknet.py

```diff
@@ -29,8 +29,10 @@ def call_gh_workflow(action,
     cmd += f"-f start_height={start_height} "
     if regions:
         cmd += f"-f location_set={regions} "
-    if validators != None:
-        cmd += f"-f validators={validators} "
+
```

Comment:
And let's choose `if x` or `if x != None` for consistency everywhere. I personally don't know what is better